Mathlib/RingTheory/Congruence/Defs.lean

```diff
@@ -129,6 +129,15 @@ theorem ext' {c d : RingCon R} (H : ⇑c = ⇑d) : c = d := DFunLike.coe_injecti
 theorem ext {c d : RingCon R} (H : ∀ x y, c x y ↔ d x y) : c = d :=
   ext' <| by ext; apply H
 
+/-- The map sending a ring congruence relation to its underlying equivalence
+relation is injective. -/
+theorem toSetoid_inj {c d : RingCon R} (H : c.toSetoid = d.toSetoid) : c = d :=
+  ext <| Setoid.ext_iff.1 H
```

Comment:
According to the [naming convention](https://leanprover-community.github.io/contribute/naming.html#injectivity), `inj` is used for lemmas of the form `f x = f y ↔ x = y`. I think this lemma would be better called `ext''`.

Reply:
OK.